### PR TITLE
Cosmos emulator change

### PR DESCRIPTION
**Pull request by g2vinay**

### CODE REVIEW

File: sdk/keyvault/ci.yml [hunk 1]
```diff
@@ -51,3 +51,6 @@ stages:
         - name: azure-keyvault-secrets
           safeName: azurekeyvaultsecrets
           stagingProfileId: 88192f04117501
+        - name: azure-keyvault-certificates
```
Comment (mitchdenny) [inline sdk/keyvault/ci.yml]: Any particular reason this is in this PR?

**Comment by mitchdenny** [comment]
> _No description provided._

:)

File: sdk/keyvault/ci.yml (hunk 1 above)
Comment (g2vinay) [reply sdk/keyvault/ci.yml]: Nope, it was accidental, removing it.

**Comment by g2vinay** [comment]
That's impressive, CI passed. :)

**Comment by mitchdenny** [comment]
Quick - merge it before it breaks :)

File: eng/pipelines/templates/variables/globals.yml [hunk 2]
```diff
@@ -4,4 +4,4 @@ variables:
   MemoryOptions: '-Xmx3072m'
   #Agent.Source.Git.ShallowFetchDepth: 1
   skipComponentGovernanceDetection: true  
-  EmulatorMsiUrl: 'https://acpedaily1.blob.core.windows.net/emulator/azure-cosmos-emulator.msi'
+  EmulatorMsiUrl: 'https://aka.ms/cosmosdb-emulator'
```
Comment (weshaggard) [inline eng/pipelines/templates/variables/globals.yml]: For my information what is the primary difference here? Are we switching from the daily to a official public version of the emulator? I guess the main question is what emulator is published at `https://aka.ms/cosmosdb-emulator`.

Comment (danieljurek) [reply eng/pipelines/templates/variables/globals.yml]: https://aka.ms/cosmosdb-emulator points to the public emulator and it is the emulator against which we have been gating changes for other languages (JS and Python).

**Pull request merged**